IDataAccess.cs

```diff
@@ -7,8 +7,17 @@
 
 namespace hotel
 {
-     public interface IDataAccess
+    public interface IDataAccess
     {
         int SaveCustomer(CustomerDto customerDto);
+
+        List<CustomerDto> MapperShowInfo();
+        List<Customer> GetCustomersInfo();
```

Comment:
Bu `Customer` in veritabanindaki hali ise buraya gelmesin. CustomerDto ile hallet butun islerini.